lib/wallet/rpc.js

```diff
@@ -1368,7 +1361,7 @@ class RPC extends RPCBase {
     if (help || args.length < 2 || args.length > 5) {
       throw new RPCError(errs.MISC_ERROR,
         'sendmany "fromaccount" {"address":amount,...}'
-        + ' ( minconf "comment" ["address",...] )');
+        + ' ( minconf "comment" subtractfee )');
```

Comment:
So this looks like a difference between `bcoin` and `bitcoind`. In the case of`bitcoind` it takes an array and will subtract the fee from those output addresses. In the case of `bcoin` it attempts to subtract from all outputs equally. So this change is correct, however if it's desired to subtract from specific output addresses, that functionality would need to be added, and this document updated.

Reply:
The `bool` behavior is documented at https://bcoin.io/api-docs/#sendmany

Reply:
Okay, also sounds like behavior with a previous version of `bitcoind` had these arguments.